pkg/apis/apps/v1alpha1/apimanagerbackup_types.go

```diff
@@ -0,0 +1,156 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+
+// APIManagerBackupSpec defines the desired state of APIManagerBackup
+// +k8s:openapi-gen=true
+type APIManagerBackupSpec struct {
+	// Backup data destination configuration
+	BackupDestination APIManagerBackupDestination `json:"backupDestination"`
+}
+
+// APIManagerBackupDestination defines the backup data destination
+// configurability. It is a union type. Only one of the fields can be
+// set
+type APIManagerBackupDestination struct {
+	// PersistentVolumeClaim as backup data destination configuration
+	// +optional
+	PersistentVolumeClaim *PersistentVolumeClaimBackupDestination `json:"persistentVolumeClaim,omitempty"`
```

Comment:
why not actual `PersistentVolumeClaim`

Reply:
Only a subset of functionality is provided